core/src/main/java/com/bloxbean/cardano/yaci/core/protocol/txsubmission/serializers/TxSubmissionMessagesSerializers.java

```diff
@@ -72,14 +72,26 @@ public byte[] serialize(ReplyTxIds replyTxIds) {
             Array array = new Array();
             array.add(new UnsignedInteger(1));
 
-            Map map = new Map();
-            if (replyTxIds.getTxIdAndSizeMap() != null) {
-                replyTxIds.getTxIdAndSizeMap().forEach((id, size) -> {
-                      map.put(new ByteString(HexUtil.decodeHexString(id)), new UnsignedInteger(size));
-                });
-            }
+//            Map map = new Map();
+//            if (replyTxIds.getTxIdAndSizeMap() != null) {
+//                replyTxIds.getTxIdAndSizeMap().forEach((id, size) -> {
+//                      map.put(new ByteString(HexUtil.decodeHexString(id)), new UnsignedInteger(size));
+//                });
+//            }
+//
+//            array.add(map);
+
+            var pairs = new Array();
+
+            replyTxIds.getTxIdAndSizeMap().forEach((id, size) -> {
+                var pair = new Array();
+                pair.add(new ByteString(HexUtil.decodeHexString(id)));
+                pair.add(new UnsignedInteger(size));
+                pairs.add(pair);
+            });
+
+            array.add(pairs);
```

Comment:
hey @satran004 quick one for you. 

I think the usage of Map here is not correct, if you check the spec it looks like the serialization expects just a list of lists, where the inner list is what it's called a pair, i.e. a list of just 2 items: id and bytes.
<img width="810" alt="Screenshot 2024-03-30 at 21 55 11" src="https://github.com/bloxbean/yaci/assets/702606/24d96307-e184-41db-bd65-911505c47364">

Unfortunately I still get the server to reset connection as soon as the agent replies.

This is an example of a serialized `ReplyTxIds` object:

`820181825820a11b541e13e3497ff88e85ef8f86ee12a26166506a4a89c1aacc05983eb9b4b018db`